src/patternfly/demos/LoginPage/examples/login-page-simple-example.hbs

```diff
@@ -12,20 +12,109 @@
     {{#> login-main}}
       {{#> login-box}}
         {{#> login-box-header}}
-          {{#> title titleType="h1" title--modifier="pf-m-3xl"}}
-            Login box title
+          {{#> title titleType="h1" title--modifier="pf-m-3xl pf-u-mb-md"}}
+            Sign in to your account
           {{/title}}
+          {{#> display display--type="inline-flex" display--modifier="pf-u-align-items-center"}}
+            {{#> dropdown id="dropdown-select" dropdown--IsSelect="true" dropdown--ItemIsSelected="true" dropdown--HasToggleIcon="true"}}
+              English
+            {{/dropdown}}
+            <p class="pf-u-ml-md">Need an account? <a href="https://www.patternfly.org/">Sign up.</a></p>
```

Comment:
I'm seeing a margin-left of 16px (dark green) in the [Login Spec](https://drive.google.com/file/d/1ElopIvkE-RDeqTpJQ2sCXjzXaIbhRafT/view) which would be md rather than lg, right?

Reply:
Yep, you're right. Disregard